src/main/scala/format/Forsyth.scala

```diff
@@ -214,7 +214,7 @@ object Forsyth {
       case _ => ""
     }
 
-  implicit private val posOrdering = Ordering.by[Pos, File](_.file)
+  implicit private val posOrdering: Ordering[Pos] = Ordering.by[Pos, File](_.file)
```

Comment:
given instead of implicit

Reply:
I didn't see how to get a simple `given` expression using `Ordering.by` here

Reply:
you can use it like this: `given Ordering[Pos] = Ordering.by[Pos, File](_.file)`. The point is `implicit` will be removed from future version of Scala 3.

Reply:
oh, of course :man_facepalming: thx